packages/dataviews/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - Add `label-position-side` classes to labels in the form field layouts. Ensure that labels in the panel view do not align center, and that all side labels are center aligned.
 - Allow readonly fields in DataForm when `readOnly` is set to `true`.
 - Adjust the padding when the component is placed inside a `Card`.
+- Introduce `perPageSizes` to control the available sizes of the items per page
```

Comment:
I'd love if we document this new prop in the README.

Reply:
Done by https://github.com/WordPress/gutenberg/pull/70604/commits/831aa1e62ed75b4262c7de9fcaa7d21b267ede50.